terraform/main.tf

```diff
@@ -0,0 +1,25 @@
+resource "aws_instance" "ec2_instance" {
+  ami                         = var.ami_id
+  subnet_id                   = var.subnet_id
+  instance_type               = var.instance_type
+  associate_public_ip_address = true
+  vpc_security_group_ids = ["sg-0b58f156501701642"]
+  key_name                    = "papila-intern"
+  iam_instance_profile = "intern-ujwal"
```

Comment:
create your own iam roles and do not hard code the variables like:
1. security group
2. iam role.